apps/site/assets/ts/schedule/__tests__/ScheduleTableTest.tsx

```diff
@@ -8,8 +8,7 @@ import { ServiceSchedule } from "../components/__schedule.js";
 import { ServiceWithServiceDate } from "../../__v3api";
 
 const services: ServiceWithServiceDate[] = [{"valid_days":[1,2,3,4,5],"typicality":"typical_service","type":"weekday","start_date":"2019-07-02","service_date":"2019-07-09","removed_dates_notes":{"2019-07-04":"Independence Day"},"removed_dates":["2019-07-04"],"name":"Weekday","id":"BUS319-O-Wdy-02","end_date":"2019-08-30","description":"Weekday schedule","added_dates_notes":{},"added_dates":[]},{"valid_days":[6],"typicality":"typical_service","type":"saturday","start_date":"2019-07-06","service_date":"2019-07-09","removed_dates_notes":{},"removed_dates":[],"name":"Saturday","id":"BUS319-P-Sa-02","end_date":"2019-08-31","description":"Saturday schedule","added_dates_notes":{},"added_dates":[]},{"valid_days":[7],"typicality":"typical_service","type":"sunday","start_date":"2019-07-07","service_date":"2019-07-09","removed_dates_notes":{},"removed_dates":[],"name":"Sunday","id":"BUS319-Q-Su-02","end_date":"2019-08-25","description":"Sunday schedule","added_dates_notes":{},"added_dates":[]}] as ServiceWithServiceDate[];
-// @ts-ignore
-const serviceSchedules: ServiceSchedule = serviceData;
+const serviceSchedules: ServiceSchedule = serviceData as ServiceSchedule;
```

Comment:
this is crazy awkward looking... but I'm assuming ts is forcing this notation?

Reply:
TS is forcing it because it's coming from JSON and it expects a specific type of key for the map. I'd get away with just one of the coercions otherwise.

```Type '{}' is not assignable to type 'ServiceSchedule'.
  Index signature is missing in type '{}'```

I was seeing that in VSCode though without both and it appears it's using a newer version of typescript than our setup (and thus jest). I think since this is in the test we can leave it as is.